package.json

```diff
@@ -17,6 +17,9 @@
   "files": [
     "lib/**/*"
   ],
+  "engines": {
+    "node": ">=12"
+  },
```

Comment:
@devanshutanna Let's bump this to >= 16.

Reply:
Ok 👍